managed/services/management/postgresql.go

```diff
@@ -119,6 +123,13 @@ func (s *PostgreSQLService) Add(ctx context.Context, req *managementpb.AddPostgr
 			if err = s.sib.GetInfoFromService(ctx, tx.Querier, service, row); err != nil {
 				return err
 			}
+
+			// In case of not available PGSM extension it is switch to PGSS.
+			if req.QanPostgresqlPgstatmonitorAgent && row.PostgreSQLOptions != nil && row.PostgreSQLOptions.PGSMVersion == "" {
+				s.l.Warningln("Extension pg_stat_monitor is not available. Switching to pg_stats_statements.")
```

Comment:
what do you think, should we show warning in pmm-admin?

Reply:
The ticket says `pg_stats_statements` should be the default, so we probably shouldn't even mention that `pg_stat_monitor` is unavailable - it's not something we event want to distract the user with.

Reply:
Actually behaviour implemented by @JiriCtvrtka in this PR is what we wanted initially in [PMM-10974](https://perconadev.atlassian.net/browse/PMM-10974)

Reply:
Right now I am updating ticket to be aligned with old one PMM-10974. 

About warning I would prefer warning (maybe info) in PMM server to dont distract user. Users with PGSM will have it and PGSS users not even notice until they take a look into server logs.

Reply:
Well, I'm looking at the commit statements - it says `PMM-12872` eveyrwhere, not `PMM-10974`. Not sure really what to look at :)

Reply:
Ticket is updated. So please take a look if it is aligned now.